mgmt_audit_project/models/nonconformity.py

```diff
@@ -10,3 +10,12 @@ class MgmtNonconformity(models.Model):
 
     name = fields.Char(required=True)
     task_ids = fields.One2many("project.task", "nonconformity_id")
+
+    @api.model
+    def _get_default_project_id(self):
+        project = self.env.ref("mgmt_audit_project.mgmt_project", raise_if_not_found=False)
+        if project:
+            return project.id
+        return self.env['project.project'].search([], limit=1).id
```

Comment:
Return the first project is wrong. It should return `False`.

Reply:
@janikvonrotz like this?
```
        def _get_default_project_id(self):
        project = self.env.ref("mgmt_audit_project.mgmt_project", raise_if_not_found=false)
        if project:
            return project.id
        return false
```